### DISCUSSION TITLE
Watch and replenish agent transaction signing account(s)

### DISCUSSION THREAD

**Discussion by barbaraliau** [celo-org/optics-monorepo#640] [Ideas]
We don't want to have to manually check if the accounts have enough funds to sign. With each new network that is added, we need to make sure we have funds for that.

Something to consider and start making a plan around to make sure we don't run out of moneys.

**Possible ways (some good, some bad)**
- CRON job
- Server to watch
- Dashboard(?)
- Somebody check everyday

**Comment by yourbuddyconner** [comment]
As mentioned in #645, we will be implementing gauges for the wallet balances in-agent which means we will have a quick and easy source of data for wallet balances. We can easily create alerts for these when balances dip below a threshold.
